### PR TITLE
docker-logs: add page

### PR DESCRIPTION
**Pull request by hjnowakowski**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

fyi, [docker](https://github.com/tldr-pages/tldr/blob/master/pages/common/docker.md) page already covers `logs` command, however gives only one example. In this PR I described `docker logs` in the more detailed manner.

### CODE REVIEW

File: pages/common/docker-logs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Print last {{num}} lines:
+
+`docker logs {{container_name}} --tail {{num}}`
+
+- Print logs and append them with timestamps:
+
+`docker logs -t {{container_name}}`
+
+- Print logs from a certain point in time of container execution (i.e. 23m, 10s, 2013-01-02T13:23:37):
```
Comment (zdroid) [inline pages/common/docker-logs.md]: Command is missing here.

**Comment by hjnowakowski** [comment]
sorry for this mess, I committed as wrong user 😤
should be good now

File: pages/common/docker-logs.md (hunk 1 above)
Comment (hjnowakowski) [reply pages/common/docker-logs.md]: added

**Comment by zdroid** [comment]
All good for me now.

I am unaware if there is an official standard for `{{bracket_names}}`, so that may be the potential problem with other reviewers. For example, maybe it's better to have `{{container name}}` instead of `{{container_name}}`.

File: pages/common/docker-logs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# docker logs
+
+> Print container logs.
+> More information: https://docs.docker.com/engine/reference/commandline/logs/.
```
Comment (pxgamer) [inline pages/common/docker-logs.md]:
```suggestion
> More information: <https://docs.docker.com/engine/reference/commandline/logs>.
```

File: pages/common/docker-logs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Print logs from a container:
+
+`docker logs {{container_name}}`
+
+- Print logs and follow them:
+
+`docker logs -f {{container_name}}`
+
+- Print last {{num}} lines:
```
Comment (pxgamer) [inline pages/common/docker-logs.md]: I'd probably change this example to use the hard coded example, i.e.
```suggestion
- Print the last 5 lines:
```

And `--tail {{5}}` in the example.

Comment (zdroid) [reply pages/common/docker-logs.md]: We definitely need to write an official style guide for things like this. :smile:

Comment (pxgamer) [reply pages/common/docker-logs.md]: Feel free to add it to the existing style guide. :+1:

Comment (zdroid) [reply pages/common/docker-logs.md]: You're right, I missed that point. :slightly_smiling_face:

Comment (hjnowakowski) [reply pages/common/docker-logs.md]: done ✅

**Pull request merged**